### PR TITLE
TTNN tests for bf8 with mk tiled scalar

### PR DESCRIPTION
**Pull request by umadevimcw**
- Updated the mk_tiled_scalar module to support bf8 to fix datatype issue for ttnn tests

### CODE REVIEW

**Review by tt-aho — APPROVED** [review]

File: tests/ttnn/python_api_testing/non_working_unit_tests/grayskull/test_eltwise_polygamma.py [hunk 1]
```diff
@@ -42,7 +42,7 @@ def run_eltwise_polygamma_tests(
 
     assert len(tt_result.shape) == len(ref_value.shape)
     assert tt_result.shape == ref_value.shape
-    assert_with_pcc(ref_value, tt_result, 0.99)
+    assert_with_pcc(ref_value, tt_result, 0.98)
```
Comment (tt-aho) [inline tests/ttnn/python_api_testing/non_working_unit_tests/grayskull/test_eltwise_polygamma.py:45]: Is this due to making the scalar tile bfp8?

I have also added mixed precision support to bcast, so these test cases should also work if the scalar tile is bfloat16 now (once #8627 is merged)

Comment (umadevimcw) [reply tests/ttnn/python_api_testing/non_working_unit_tests/grayskull/test_eltwise_polygamma.py:45]: Yes @tt-aho

Comment (npetrovic-tenstorrent) [inline tests/ttnn/python_api_testing/non_working_unit_tests/grayskull/test_eltwise_polygamma.py:45]: Are we allowed to lower PCC this way? We are usually not doing it this way for other operations.

File: tests/ttnn/python_api_testing/non_working_unit_tests/grayskull/test_eltwise_threshold.py [hunk 2]
```diff
@@ -45,7 +45,7 @@ def run_eltwise_threshold_tests(
     assert tt_result.shape == ref_value.shape
 
     # compare tt and golden outputs
-    success, pcc_value = comp_pcc(ref_value, tt_result)
+    success, pcc_value = comp_pcc(ref_value, tt_result, pcc=0.97)
```
Comment (npetrovic-tenstorrent) [inline tests/ttnn/python_api_testing/non_working_unit_tests/grayskull/test_eltwise_threshold.py:48]: Are we allowed to lower PCC value in unit tests?

**Review by eyonland — APPROVED** [review]

**Review by xanderchin — APPROVED** [review]

**Review by arakhmati — APPROVED** [review]

**Pull request merged**